.eslint-ci.config.js

```diff
@@ -21,12 +24,14 @@ module.exports = {
       'error',
       {
         devDependencies: [
+          'jest*.[jt]s',
           'packages/*/*/{__helpers__,__stories__,__tests__,__tests-ssr__}/**/*',
           'packages/private/bundle-size/webpack.config.js',
           'packages/private/demo/{scripts,src}/**/*',
           'scripts/**/*',
         ],
         optionalDependencies: false,
+        packageDir: [__dirname, ...getPackageDirectories()],
```

Comment:
What does this achieve? As far as I know, ESLint already finds our packages correctly.